### PR TITLE
Fix minor bugs GDV-214, 216, 220, 221, 222

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: components/site-navigation.vue [hunk 1]
```diff
@@ -138,7 +154,8 @@ export default {
 
 .site-navigation__list-item--active:hover,
 .site-navigation__list-item--active:active,
-.site-navigation__list-item--active:focus {
+.site-navigation__list-item--active:focus,
```
Comment (SiggyF) [inline components/site-navigation.vue]: Is `site-navigation__list-item--active` still used?

File: components/v-mapbox-components/v-mapbox-vector-layer.vue [hunk 2]
```diff
@@ -34,67 +33,64 @@ export default {
...
     },
     updateMap() {
-      this.layers.forEach((newLayer, index) => {
-        if (this.map.getLayer(this.layers[index].id)) {
-          this.map.setFilter(this.layers[index].id, newLayer.filter)
-        } else {
-          this.addToMap()
-        }
-      })
+      if (this.map.getLayer(this.layer.id)) {
```
Comment (SiggyF) [inline components/v-mapbox-components/v-mapbox-vector-layer.vue]: why is map added in updateMap?

File: components/v-mapbox-components/v-mapbox-vector-layer.vue [hunk 3]
```diff
@@ -34,67 +33,64 @@ export default {
...
-          setTimeout(() => {
-            this.$emit('select-locations', {
-              features,
-              geometry: features[0].geometry,
-            })
-          }, duration)
+        // prettier-ignore
+        // the timeseries panel is max 600px wide otherwise the half of the screen
+        const visibleMapWidth = clientWidth > 1200
+          ? (clientWidth - 600) * 0.25
```
Comment (SiggyF) [inline components/v-mapbox-components/v-mapbox-vector-layer.vue]: why not in css?

File: components/site-navigation.vue [hunk 4]
```diff
@@ -7,17 +7,23 @@
   >
     <ul class="site-navigation__list">
       <li v-for="(theme, key) in getThemes" :key="key">
-        <div
-          class="site-navigation__list-item site-navigation__list-item--active"
-        >
-          <UiButtonIcon @click="changeTheme(theme.id)">
+        <div class="site-navigation__list-item site-navigation__list-item">
+          <UiButtonIcon
+            :class="{ 'nav-active': checkActive(theme.id) }"
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply components/site-navigation.vue]: went for: site-navigation__list-item--active

File: components/site-navigation.vue (hunk 1 above)
Comment (CindyvdVries) [reply components/site-navigation.vue]: now it is again

**Pull request merged**